urlshort/handler.go

```diff
@@ -32,6 +46,32 @@ func MapHandler(pathsToUrls map[string]string, fallback http.Handler) http.Handl
 // See MapHandler to create a similar http.HandlerFunc via
 // a mapping of paths to urls.
 func YAMLHandler(yml []byte, fallback http.Handler) (http.HandlerFunc, error) {
-	// TODO: Implement this...
-	return nil, nil
+	// Parse yml to a map
+	pathsToUrls := make(map[string]string)
+	seq := make([]Item, 0)
```

Comment:
You don't need to make a slicer since you are just passing a reference (although not super relevant here, slices are zero safe meaning that you can range over and append to a nil slice).

Reply:
Thanks